verkle-trie/src/errors.rs

```diff
@@ -1,23 +1,17 @@
 use ark_serialize::SerializationError;
 use thiserror::Error;
 
-// A Wrapper Type for all errors that can occur within the Verkle Library
-// Provides Single Error Enum for consumers of the library to match against
 // Right now there are lots of unwraps which are immediately switched to Results, but in the future
 // We likely be moved back to unwraps with safety comments
-#[derive(Error, Debug)]
-pub enum VerkleError {
-    #[error("Issue Occured Converting Type to Bytes")]
-    SerializationError(#[from] SerializationError),
-    #[error("Precomputed Points Exist Already")]
-    PrecomputedPointsFileExists,
-    #[error("Unable to Create Precomputed Points File")]
-    CannotCreatePrecomputedPoints(std::io::Error),
-    #[error("Precomputed Lagrage Points File Couldn't not be found")]
-    PrecomputedPointsNotFound,
-    #[error("Issue opening PrecomputedPointsFile")]
-    FileError(std::io::Error),
 
+#[derive(Debug, Error)]
+pub enum HintError {
```

Comment:
Can we just use std::io::Error as its the only variant at the moment and I'm not sure `HintError` is a unambiguous name

Reply:
Hmm okay I see what you are doing here, the enum is better and HintError can be bikeshedded at a later date